packages/amplify-category-geo/src/service-walkthroughs/mapWalkthrough.ts

```diff
@@ -121,14 +109,13 @@ export const mapStyleWalkthrough = async (parameters: Partial<MapParameters>): P
     const mapStyleDefault = parameters.dataProvider && parameters.mapStyleType ?
         getGeoMapStyle(parameters.dataProvider, parameters.mapStyleType) : 'VectorEsriStreets';
 
-    const mapStyleTypePrompt = {
-        type: 'list',
-        name: 'mapStyle',
-        message: `Specify the map style. Refer ${apiDocs.mapStyles}`,
-        choices: mapStyleChoices,
-        default: mapStyleDefault
-    };
-    return getMapStyleComponents((await inquirer.prompt([mapStyleTypePrompt])).mapStyle as MapStyle);
+    const mapStyleDefaultIndex = mapStyleChoices.findIndex(item => item.value === mapStyleDefault);
```

Comment:
This is `es6` JS function. Hope that is supported?

Reply:
Looks like it's supported since node 4 so we're good